### PR TITLE
Spec

### PR DESCRIPTION
**Pull request by jamesprinc3**
Fixes issue #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,30 @@
...
+
+Students can:
+* Insert all details which companies require so that they don’t have to fill any other forms
+* Exactly which fields is up for discussion
+* Companies usually hire for multiple positions/countries. How will we handle that?
+* Select which companies they wish to intern/placement/grad at
+* Give permission to share personal information (ie. CV, email, mobile, LinkedIn) with sponsors (very important)
+* Give permission to allow other students to see their profile
+* Filter sponsors based on what the sponsors are looking for (eg a fresher should see that Company X doesn’t take freshers)
+* Track their applications
```
Comment (plietar) [inline README.md:42]: Should this be a tool that can be use to track applications to other companies that aren't sponsors ?

**Comment by plietar** [comment]
Looks good to me.

Thoughts on using GitHub's projects to track implementation progress ?

**Review by plietar — APPROVED** [review]

**Comment by egnwd** [comment]
@plietar I like github projects/milestones combo - works pretty well

**Comment by egnwd** [comment]
I'll create some issues off the back of this then we can add context and details as and when

**Pull request merged**